two.js

```diff
@@ -0,0 +1,135 @@
+// Elliott Katane - 6214821
+
+let myForm = document.getElementById('myform');
+
+myForm.addEventListener('submit', function(e) {
```

Comment:
There are 2 different event listeners for the submit action on `myForm` making it a bit unclear before reading the content of the function what differs from one to the other. 

I guess what could be done is to create named functions for both of them, something like 

```javascript
myForm.addEventListener('submit', validateName())


function validateName(e) {
  // put the rest of the function here
}
```

Reply:
And the same thing could be done for the email validation with a `validateEmail() `function.